File: 7class/Tema6/ex5.py
#include<iostream>
using namespace std;
int main() 
{ 
    int a,b,c;
    cin >> a >> b >> c;
    if((a+b>c)&&(a+c>b)&&(b+c>a)) 
     {
         if ((a*a==b*b+c*c)||(b*b==a*a+c*c)||(c*c==a*a+b*b)) cout << "Rectangulal";
         else if ((a*a>b*b+c*c)||(b*b>a*a+c*c)||(c*c>a*a+b*b)) cout << "Obtuse";
              else cout << "Sharp";
     }
    else cout << "Invalid";
}

File: 7class/ex5.cpp
using namespace std;

int main() {

  int a, b, i;
  cin >> a >> b;
  
  if (a<b)
    {
     for (i=a; i<=b; i++)
      {
        cout << i;
        if (i<b) cout << " ";
      }
    } 
  else
   {
     for (i=a; i>=b; i--)
      {
        cout << i;
        if (i>b) cout << " ";
      }  
   }
}